### PR TITLE
[Check-in] Introduce new attribute to replace the deprecated ones

### PR DESCRIPTION
**Pull request by nikosev**
# Summary

This PR introduces the following changes:

- Add `distinguishedName` in the deprecated attributes table of the SAML migration guide
- Remove `eduPersonScopedAffiliation` attribute from "Affiliation" attribute description
- Add description for "Subject DN" attribute
- Add description for "Issuer DN" attribute

### CODE REVIEW

File: content/en/providers/check-in/sp/_index.md [hunk 1]
```diff
@@ -1919,20 +1920,20 @@ connected to Check-in.
...
-|             **example** | <ul><li>`member@example.org`</li><li>`faculty@example.org`</li></ul>                                                                                       |
-|               **notes** | Service Providers are encouraged to validate the scope of this attribute                                                                                   |
-|              **status** | Stable                                                                                                                                                     |
+|          attribute name | Affiliation                                                              |
+| ----------------------: | :----------------------------------------------------------------------- |
+|         **description** | The user's affiliation within a particular security domain (scope)       |
+|   **SAML Attribute(s)** | `urn:oid:1.3.6.1.4.1.25178.4.1.11` (voPersonExternalAffiliation)         |
+|          **OIDC scope** | `eduperson_scoped_affiliation`                                           |
+|       **OIDC claim(s)** | `eduperson_scoped_affiliation`                                           |
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md]: voperson_external_affiliation

File: content/en/providers/check-in/sp/_index.md [hunk 2]
```diff
@@ -1919,20 +1920,20 @@ connected to Check-in.
...
+| **OIDC claim location** | <ul><li>UserInfo Endpoint</li><li>Introspection Endpoint</li></ul>       |
+|              **origin** | Check-in assigns this attribute on user registration                     |
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md]: When available, this information originates from the user's authenticating Identity Provider

File: content/en/providers/check-in/sp/_index.md [hunk 3]
```diff
@@ -2056,6 +2057,41 @@ connected to Check-in.
...
 <!-- textlint-enable -->
+
+### 17. Subject Distinguished Name (DN) of user's X.509 certificate
+
+|          attribute name | Subject Distinguished Name (DN) of user's X.509 certificate                |
+| ----------------------: | :------------------------------------------------------------------------- |
+|         **description** | The Subject Distinguished Name of an X.509 certificate held by the person. |
+|   **SAML Attribute(s)** | `1.3.6.1.4.1.25178.4.1.3` (voPersonCertificateDN)                          |
+|          **OIDC scope** | <ul><li>`voperson_certificate_dn`</li><li>`aarc`</li></ul>                 |
+|       **OIDC claim(s)** | `voperson_certificate_dn`                                                  |
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:2069]: Change to `voperson_certificate`

File: content/en/providers/check-in/sp/_index.md [hunk 4]
```diff
@@ -2056,6 +2057,41 @@ connected to Check-in.
...
+|              **status** | Experimental                                                               |
+
+### 18. Issuer Distinguished Name (DN) of user's X.509 certificate
+
+|          attribute name | Issuer Distinguished Name (DN) of user's X.509 certificate              |
+| ----------------------: | :---------------------------------------------------------------------- |
+|         **description** | The Subject Distinguished Name of the X.509 certificate issuer.         |
+|   **SAML Attribute(s)** | `1.3.6.1.4.1.25178.4.1.4` (voPersonCertificateIssuerDN)                 |
+|          **OIDC scope** | <ul><li>`voperson_certificate_issuer_dn`</li><li>`aarc`</li></ul>       |
+|       **OIDC claim(s)** | `voperson_certificate_issuer_dn`                                        |
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md]: Change to `voperson_certificate`

File: content/en/providers/check-in/sp/_index.md (hunk 3 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md:2069]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 1 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 2 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md]: done

File: content/en/providers/check-in/sp/_index.md (hunk 4 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md]: Done

File: content/en/providers/check-in/sp/_index.md [hunk 5]
```diff
@@ -531,15 +532,16 @@ UserInfo Endpoint:
...
+| `voperson_certificate`          | <ul><li>`voperson_certificate_dn`</li><li>`voperson_certificate_issuer_dn`</li></ul>                                                                                                                                                                                                                                 |
+| `voperson_external_affiliation` | `voperson_external_affiliation`                                                                                                                                                                                                                                                                                      |
+| `voperson_id`                   | `voperson_id`                                                                                                                                                                                                                                                                                                        |
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md]: Move voperson_id after openid to promote its use

File: content/en/providers/check-in/sp/_index.md [hunk 6]
```diff
@@ -531,15 +532,16 @@ UserInfo Endpoint:
...
+| Scope                           | Claims                                                                                                                                                                                                                                                                                                               |
+| ------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `openid`                        | <ul><li>`sub`</li><li>`eduperson_assurance`</li></ul>                                                                                                                                                                                                                                                                |
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md]: I suggest we remove eduperson_assurance from here (it's already mentioned in the user attributes/claims section

Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 5 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md]: done

**Pull request closed without merging**